jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -333,12 +334,23 @@ object ConverterUtils extends Logging {
   final val ALIAS = "alias"
   final val IS_NOT_NULL = "is_not_null"
   final val IS_NULL = "is_null"
-  // Datetime
+  final val NOT = "not"
+
+  // String
+  final val ASCII = "ascii"
   final val EXTRACT = "extract"
   final val ENDS_WITH = "ends_with"
+  final val CONCAT = "concat"
   final val CONTAINS = "contains"
-  final val NOT = "not"
+  final val INSTR = "instr"
+  final val LENGTH = "length"
+  final val LOWER = "lower"
+  final val UPPER = "upper"
+  final val REPLACE = "replace"
+  final val SPLIT = "split"
```

Comment:
Please check whether these functions are already defined in Substrait yaml (https://github.com/substrait-io/substrait/tree/main/extensions). If yes, need to use the same name as its definition, if not, you can open PR to add.